mysql/infoschema.go

```diff
@@ -375,6 +380,47 @@ func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (forei
 	return foreignKeys, nil
 }
 
+// getIndexes return list all the indexes.
+func getIndexes(conv *internal.Conv, db *sql.DB, table schemaAndName) (indexes []schema.Index, err error) {
+	q := `SELECT DISTINCT INDEX_NAME,COLUMN_NAME,SEQ_IN_INDEX,COLLATION,NON_UNIQUE
+		FROM INFORMATION_SCHEMA.STATISTICS 
+		WHERE TABLE_SCHEMA = ?
+			AND TABLE_NAME = ?
+			AND INDEX_NAME != 'PRIMARY' 
+		ORDER BY INDEX_NAME, SEQ_IN_INDEX;`
+	rows, err := db.Query(q, table.schema, table.name)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var name, col, seq, collation, nonUnique string
+	m := make(map[string][]schema.Key)
```

Comment:
considering using a more descriptive variable name for this map since it's difficult to read in the current form

also, rather than create maps for keys i.e., m and for unique constraint i.e., unique, it might be easier to create a map from indexKey name to schema.Key, and populate it within rows.Next() loop; this way you may be able to catch 

at the very end before returning you can then create an array from map with sorted indexKey names

Reply:
I am also wondering whether indexKey names are unique in mysql db globally or per table?

Reply:
IndexKey names are unique per table only in mysql, while in postgres it has to be unique globally

Reply:
updated variable names and updated approach as suggested for populating indexes.

Reply:
updated variable names as part of other comments